pkg/subctl/cmd/root.go

```diff
@@ -100,6 +101,14 @@ func getClients(config *rest.Config) (dynamic.Interface, kubernetes.Interface, e
 	return dynClient, clientSet, nil
 }
 
+func getClusterName(rawConfig clientcmdapi.Config) *string {
+	context, ok := rawConfig.Contexts[rawConfig.CurrentContext]
```

Comment:
I wonder if we may check with the broker that we are not repeating the cluster ID (separate PR).

Reply:
I was thinking... let's add this to the operator... but ... it's not possible. We don't have access to the context name at that time.